spec/lib/dispatch-rider/logging/text_formatter_spec.rb

```diff
@@ -1,7 +1,11 @@
 require 'spec_helper'
 
 describe DispatchRider::Logging::TextFormatter do
-  let(:message) { DispatchRider::Message.new(subject: 'test', body: 'test_handler') }
+  let(:fs_message) { DispatchRider::Message.new(subject: 'test', body: { 'key' => 'value', 'guid' => 123 }) }
+  let(:item) { double }
```

Comment:
please name your doubles, otherwise errors related to them will be unhelpful .. `let(:item) { double :item }`